jsettlers.common/resources/buildings/big_livinghouse.xml

```diff
@@ -118,30 +116,26 @@
 	<blocked dx="-5" dy="-3" block="false" />
 	<blocked dx="-5" dy="-2" block="false" />
 	<blocked dx="-5" dy="-1" block="false" />
-	<blocked dx="-4" dy="1" block="false" />
```

Comment:
I know, but I checked that there is still a closed protected area around the building.